Makefile

```diff
@@ -121,4 +163,7 @@ dist: check $(OTF) $(PDF) $(OPDF)
 	@zip -rq $(DIST).zip $(DIST)
 
 clean:
-	@rm -rf $(DIST) $(DIST).zip $(CHK) $(MIS) $(DUP) $(NRM) $(OTF) $(PDF)
+	@rm -rf $(DIST) $(DIST).zip $(CHK) $(MIS) $(DUP) $(FEA) $(NRM) $(LNT)
+
+cleaner: clean
```

Comment:
`make clean` should remove any generated file, no need for another target.

Reply:
I don't get this one. `.gitignore` and the genuinely temporary files work well together, in that the latter are ignored for purposes of diffing and committing. But with the current Makefile if you do a `make clean` and then `git commit -a`, you delete a bunch of files intended to be under version control. Why would you want that? Isn't there an important difference between a generated file intended to be checked in when it changes, and a generated file only ever meant to be temporary? 

I mean, I'll do it. I say what I think and I do what I'm told™. But I don't get it.

Reply:
The idea is that if I `make clean` then `make` then any generated file should be definitely generated. Ideally OTF and PDF files shouldn’t be under version control to begin with, but since font building pipeline is so fragile I keep them under version control for when I need to debug a regression and find whether it is a font or tool regression.

Reply:
As someone who fiddles around with Libertinus I want a target that leaves me with exactly the set of files under version control. 

What if I make it "less-clean" and "clean" and no one has to know about "less-clean" who doesn't peek?

Reply:
That would be `git clean -fdx`, no need for a make target :)